README.md

```
@@ -534,120 +534,263 @@ OPENCODE_ANTIGRAVITY_PID_OFFSET_ENABLED=1                 # pid_offset_enabled
-    "gemini": 0
   }
 }
 ```
 
-**Note**: This would have to be done per account if you have a multi-account setup. 
+---
+
+### Gemini 3 Models 400 Error ("Unknown name 'parameters'")
+
+**Error:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)